src/test/java/bit/javaoop/CompanyTest.java

```diff
@@ -0,0 +1,48 @@
+package bit.javaoop;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class CompanyTest {
+    @Test
+    void hireNewCeo() {
+
+        CEO newCeo = new CEO("Adam");
+        CEO oldCeo = new CEO("Ala", Arrays.asList(new Manager[]{new FixedBudgetManager("A", 1, 1), new FixedEmployeeNumberManager("B", 2, 2)}));
+        List<Manager> oldList = oldCeo.getManagersList();
+        Company company = new Company("C", oldCeo);
+        company.hireNewCeo(newCeo);
+        assertEquals("Adam", company.getCeo().getName());
+        assertEquals(oldList, company.getCeo().getManagersList());
+
+
+    }
+
+    @Test
+    void tostring() {
+
+        Company company = new Company("CCC");
+        company.hireNewCeo(new CEO("Adam"));
+        FixedBudgetManager fbm1 = new FixedBudgetManager("Adam_budget ", 3000, 900);
+        company.getCeo().hireManager(fbm1);
+        FixedBudgetManager fbm2 = new FixedBudgetManager("Ela_budget ", 30800, 77900);
+        company.getCeo().hireManager(fbm2);
+        FixedEmployeeNumberManager fenm1 = new FixedEmployeeNumberManager("Kasia_limit", 2000, 1);
```

Comment:
Zamiast takich skrótów strasznych jak fbm1 fbm2 już użyłbym managerOne managerTwo albo manager1 manager2
jak ktoś będzie chciał sprawdzić typ to najedzie sobie z controlem na zmienną:)